package.json

```diff
@@ -4,8 +4,9 @@
   "description": "A social platform for the creative at heart ",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --mode development --open --hot",
+    "start": "webpack-dev-server --mode development --hot",
```

Comment:
Any reason why you removed the open flag?

Reply:
I have added the '--open' flag back to the start script.